apps/daffio/src/app/docs/guide-docs/views/guide-view.component.ts

```diff
@@ -0,0 +1,23 @@
+import {
+  Component,
+  OnInit,
+} from '@angular/core';
+import { ActivatedRoute } from '@angular/router';
+import { Observable } from 'rxjs';
+import { map } from 'rxjs/operators';
+
+import { DaffioDoc } from '../../shared/models/doc';
+
+@Component({
+  templateUrl: './guide-view.component.html',
+})
+export class DaffioGuideViewComponent implements OnInit {
```

Comment:
After looking at the API view and the guide view, they look the same, can these be centralized instead?